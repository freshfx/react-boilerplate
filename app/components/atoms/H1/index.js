import styled from '@emotion/styled'

const H1 = styled.h1`
  font-size: 2em;
  margin-bottom: 0.25em;
`

export default H1
