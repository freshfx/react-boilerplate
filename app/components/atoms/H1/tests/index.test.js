import React from 'react'
import {render} from '@testing-library/react'

import H1 from '../index'

describe('H1', () => {
  it('should match snapshot', () => {
    const {container} = render(<H1 />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
