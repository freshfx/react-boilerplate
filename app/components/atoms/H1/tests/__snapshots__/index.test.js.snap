// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H1 should match snapshot 1`] = `
.emotion-0 {
  font-size: 2em;
  margin-bottom: 0.25em;
}

<h1
  class="emotion-0 emotion-1"
/>
`;
